downstream/modules/platform/proc-installing-the-ansible-builder-rpm.adoc

```diff
@@ -2,19 +2,25 @@
 
 = Installing the ansible-builder RPM
 
-.Procedure
+[role="_abstract"]
 
-. On a RHEL system, install the ansible-builder RPM. This can be done in one of several ways:
+In order to create custom {ExecEnvShort} images in a disconnected environment, you will need to have a working copy of ansible-builder installed. On a RHEL system there are several ways to do this.
+
+* Subscribe the RHEL box to a Satellite on the disconnected network.
+* Attach the Ansible Automation Platform subscription and enable the AAP repo.
```

Comment:
AAP is not an approved acronym and I think it should be replaced here unless that is the actual name of the repo.

Reply:
I don't know if that is the name of the repo or not. It was written like that by the devs, so I used it. I doubt there is anyone around now who can tell me if it is or not.